components/layout/grid/src/_settings.scss

```diff
@@ -0,0 +1,67 @@
+$lg-breakpoints: $breakpoints !default;
+$lg-column-gap: $p-l !default;
```

Comment:
please follow the naming convention https://docs.mpi-internal.com/scmspain/frontend-convergence/Sass/Variables/

Reply:
Thanks, update PR!